### PR TITLE
Fixes blueprints exploit

### LINKED ISSUE #36496: Cerulean blueprints are still usable even after they're deleted after a use.
[Round ID]: N/A I forgot and it was on citadel

[Testmerges]: N/A

[Reproduction]: Use a cerulean blueprint, don't close the window, move to another area, mark it again. Rinse repeat.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #36496

### CODE REVIEW

File: code/game/objects/items/blueprints.dm [hunk 1]
```diff
@@ -25,12 +26,16 @@
 
 /obj/item/areaeditor/Topic(href, href_list)
 	if(..())
-		return
-	if(!usr.canUseTopic(src))
+		return TRUE
+	if(!usr.canUseTopic(src) || QDELETED(src))
```
Comment (Cyberboss) [inline code/game/objects/items/blueprints.dm]: I feel like a QDELETED check should be higher up the call chain. You sure this isn't in ..() or /client/Topic()?

Comment (kevinz000) [reply code/game/objects/items/blueprints.dm]: it doesnt

Comment (vuonojenmustaturska) [reply code/game/objects/items/blueprints.dm]: #36666

**Comment by AnturK** [comment]
Remove the redundant qdeleted.

**Pull request merged**